Documentation/cmdref/cilium-agent.md

```diff
@@ -36,6 +36,7 @@ cilium-agent [flags]
       --bpf-lb-bypass-fib-lookup                             Enable FIB lookup bypass optimization for nodeport reverse NAT handling (default true)
       --bpf-lb-dev-ip-addr-inherit string                    Device name which IP addr is inherited by devices running LB BPF program (--devices)
       --bpf-lb-dsr-dispatch string                           BPF load balancing DSR dispatch method ("opt", "ipip") (default "opt")
+      --bpf-lb-enable-cluster-ip-external-access             Enable external access to ClusterIP services
```

Comment:
nit: I would name it something shorter, like `--bpf-lb-external-clusterip`

Reply:
Yep, shorter is better. Renamed.